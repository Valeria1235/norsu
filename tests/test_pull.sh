set -v

export NORSU_PATH="$PWD/pg"

# pull master
norsu pull master

# check that dir exists
if [ -e "$NORSU_PATH/.norsu/master" ]; then echo OK; fi

# purge master
norsu purge master

# check that dir has been removed
if [ ! -e "$NORSU_PATH/.norsu/master" ]; then echo OK; fi

# remove dir
rm -rf "$NORSU_PATH"
