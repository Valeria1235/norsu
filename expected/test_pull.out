
export NORSU_PATH="$PWD/pg"

# pull master
norsu pull master
Selected instance: master
	=> No work dir, choosing repo & branch
	=> Selected repo git://git.postgresql.org/git/postgresql.git
	=> Selected branch master
	=> Cloned git repo to work dir


# check that dir exists
if [ -e "$NORSU_PATH/.norsu/master" ]; then echo OK; fi
OK

# purge master
norsu purge master

# check that dir has been removed
if [ ! -e "$NORSU_PATH/.norsu/master" ]; then echo OK; fi
OK

# remove dir
rm -rf "$NORSU_PATH"
